README.md

```diff
@@ -1,37 +1,60 @@
 <div align="center">
 
 # Myria3D: Aerial Lidar HD Semantic Segmentation with Deep Learning
+# Fork adapted to train/infer on non-colorized data
```

Comment:
Can you remove the parts that are relate to your fork from this MR please, as it won't be relevant once merged